tsconfig.json

```diff
@@ -14,9 +14,16 @@
   },
   "include": [
     "./src/**/*.ts",
+    "./scripts/**/*.ts",
     "./test/**/*.ts"
   ],
   "exclude": [
     "node_modules"
-  ]
+  ],
+  "ts-node": {
+    "experimentalSpecifierResolution": "node",
+    "compilerOptions": {
+      "module": "CommonJS"
```

Comment:
Could you briefly explain why this change was needed? The whole `ts-node` object

Reply:
I would like to have different options than tsconfig.json when I want to run my scripts with ts-node. But I wouldn't like to change all tsconfig for that, so instead I used ts-node option only available for running ts-node environment.